src/components/team.js

```diff
@@ -5,7 +5,7 @@ import "../styles/global.css"
 export default function Team() {
   const { teamData } = data
   const [year, setYear] = useState(Object.keys(teamData).reverse()[1])
-  const [team, setTeam] = useState(teamData["2022"])
+  const [team, setTeam] = useState(teamData["2020"])
```

Comment:
let's keep 2022?

Reply:
You need to pull the recent changes I think, try these:
`
git checkout main
git remote add upstream https://github.com/linustribevnr/turinghut-website.git
git fetch upstream
git rebase upstream/main
git checkout "you branch name"
git rebase main
git push origin "you branch name" --force
`
After this, you'll have all the new changes made to the repo in your fork, then you can access the 2022 data.